README.md

```diff
@@ -268,7 +268,7 @@
 
 ## Installation
 
-The easiest way to install Bokeh is using the [Anaconda Python distribution](https://www.anaconda.com/what-is-anaconda/) and its included *Conda* package management system. To install Bokeh and its required dependencies, enter the following command at a Bash or Windows command prompt:
+The [Anaconda Python distribution](https://www.anaconda.com/what-is-anaconda/) and its included *Conda* package management system are the simplest way to install Bokeh. Enter the following command at a Bash or Windows command prompt to install Bokeh and its dependencies:
```

Comment:
The order of this sentence was intentional, to lead with and emphasize "easy way to install Bokeh". Pleas revert this change.

Reply:
Reverting this as well